composer.json

```diff
@@ -13,9 +13,12 @@
         "keboola/db-extractor-common": "^9.2"
     },
     "require-dev": {
-        "phpunit/phpunit": "7.*",
         "ext-pdo_pgsql": "*",
-        "aws/aws-sdk-php": "^3.18"
+        "aws/aws-sdk-php": "^3.18",
+        "jakub-onderka/php-parallel-lint": "^1.0",
+        "keboola/coding-standard": "^8.0",
```

Comment:
Coding standard se nově používá takto: 

https://github.com/keboola/component-generator/blob/master/templates/php-component/composer.json#L8

(tj. cokoli větší než minimální, vč. v 7, 8 ,9.1.3 ... )

Je to proto, aby odpadlo frcání s přepisováním aktuální verze. Prostě se jen udělá `composer update keboola/coding-standard`. Tím, že máme composer.lock v komponentách, tak to není problém.